packages/experimental/Avatar/ios/AvatarViewManager.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+import FluentUI
+
+@objc(MSFAvatarViewManager)
+class AvatarViewManager: RCTViewManager {
+	override func view()->UIView! {
+		let avatarView = AvatarView(avatarSize: .small)
+		return avatarView
+	}
+
+	override class func requiresMainQueueSetup() -> Bool {
+		return true
+	}
+
+	override func constantsToExport() -> [AnyHashable : Any]! {
+			return [
```

Comment:
nit- the spacing in this method isn't right

Reply:
Yep, there is an extra tab. I'll fix that